classify/tutorial.ipynb

```diff
@@ -723,7 +723,7 @@
             "                  cradle          50        0.38         0.8\n",
             "         crane (machine)          50        0.78        0.94\n",
             "            crash helmet          50        0.92        0.96\n",
-            "                   crate          50        0.52        0.82\n",
+            "                   create          50        0.52        0.82\n",
```

Comment:
This is probably not a correct typo fix.

Reply:
Yeah these exclude words are in the pyproject.toml settings that are missing from this PR, I need to add it!